### DISCUSSION TITLE
Welcome to OpenWHU Discussions!

### DISCUSSION THREAD

**Discussion by Crescentm** [openwhu/OpenWHU#41] [General]
## 👋 欢迎!
  为了更好的交流，我们在这里开设了一个讨论区，你可以在这里发表你的想法，包括且不限于:
  * 询问关于项目的问题.
  * 分享一些项目
  * 发表校园生活的想法
   ### 欢迎大家在这里畅所欲言

**Comment by jfang2048** [comment]
你好，请问有地理信息系统原理的么

**Reply by Crescentm** [reply]
看了下，好像没有的样子

**Comment by jfang2048** [comment]
基本是计算机的啊，测绘、遥感、地信几乎没有，希望能补充一下

**Reply by Crescentm** [reply]
/(ㄒoㄒ)/~~ 没有人捐献呀
